libs/lib_sql_handler.py

```diff
@@ -84,15 +101,50 @@ def add_to_table(self, table: str, data: Union[List[Any], Tuple[Any, ...]]) -> N
 
         self.cur.execute(db_inputStr, tuple(db_inputList))
 
-    def fetch_rows_from_table(self, table: str, column_search: List[Any]) -> Tuple[Any, ...]:
+    def multicount_rows_from_table(self, table: str, searchparms: List[List[Any]]) -> int:
 
         # Test for attack
         if table.count("\\") or table.count("'"):
             raise db_error.OperationalError("Detected SQL injection attack")
 
         # Add SELECT data
-        db_inputStr = f"SELECT * FROM '{table}' WHERE {column_search[0]} = ?"
-        db_inputList = [column_search[1]]
+        db_inputStr = f"SELECT COUNT(*) FROM '{table}' WHERE "
+
+        db_inputStr += " AND ".join([f"({i[0]} {i[2] if len(i) > 2 else '='} ?)" for i in searchparms])
+        db_inputList = [i[1] for i in searchparms]
+
+        # Execute
+        self.cur.execute(db_inputStr, tuple(db_inputList))
+
+        retval: int = tuple(self.cur.fetchall())[0][0]
+        return retval
+
+    def fetch_rows_from_table(self, table: str, search: List[Any]) -> Tuple[Any, ...]:
+
+        # Test for attack
+        if table.count("\\") or table.count("'"):
```

Comment:
you used `in` previously

Reply:
its the difference between str.__contains__ and str.count its not that much slower i cba to rewrite the rest